client/pdo/client/scripts/eservicedatabaseCLI.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+ 
+import os
+import sys
+
+import argparse
+import logging
+logger = logging.getLogger(__name__)
+
+import pdo.common.config as pconfig
+import pdo.common.logger as plogger
+
+import pdo.service_client.service_data.eservice as db
+
+import argparse
+
+def Add_entries(config):
+    """ Add entries to database. Return number of entries added"""
```

Comment:
why are you adding through a configuration file?

Reply:
there is option to create the database using the urls listed in pcontract.toml. any command line urls will override the toml contents. The Add function is resued for create and add.